app/src/main/java/org/mozilla/fenix/tabtray/TabTrayViewHolder.kt

```diff
@@ -26,16 +26,10 @@ import mozilla.components.feature.media.ext.playIfPaused
 import mozilla.components.support.base.observer.Observable
 import mozilla.components.support.images.ImageLoadRequest
 import mozilla.components.support.images.loader.ImageLoader
-import mozilla.components.support.ktx.kotlin.tryGetHostFromUrl
 import org.mozilla.fenix.R
 import org.mozilla.fenix.components.metrics.Event
 import org.mozilla.fenix.components.metrics.MetricController
-import org.mozilla.fenix.ext.components
-import org.mozilla.fenix.ext.getMediaStateForSession
-import org.mozilla.fenix.ext.increaseTapArea
-import org.mozilla.fenix.ext.removeAndDisable
-import org.mozilla.fenix.ext.removeTouchDelegate
-import org.mozilla.fenix.ext.showAndEnable
+import org.mozilla.fenix.ext.*
```

Comment:
Wildcard imports will fail our `ktlint` task.
Please rollback this change.

Reply:
I fixed the imports and squashed the branch